util/scale_run.sh

```diff
@@ -0,0 +1,230 @@
+#!/bin/bash
+
+# Run scale tests for each Standard Ref Arch size on docs page
+#   Ref: https://puppet.com/docs/pe/latest/hardware_requirements.html
+
+
+##########
+# Function Definitions
+##########
+
+# Display usage help
+usage () {
+    echo "$0 [-h] [-d] PE_VERSION -- Run scale testing on specified PE_VERSION"
+    echo
+    echo "where:"
+    echo "            -h  show this help text"
+    echo "            -d  debug mode: prints ENV values and commands to be executed, but does not run anything"
+    echo "    PE_VERSION  in semver notation (e.g 2019.1.0)"
+    echo
+}
+
+# Debug function to validate that bash subshells inherit
+# expected environment variables.
+function echo_env () {
+    echo "============================================
+In subshell for $test $run_type round $i
+cmd=$cmd
+BEAKER_INSTALL_TYPE=$BEAKER_INSTALL_TYPE
+BEAKER_PE_DIR=$BEAKER_PE_DIR
+BEAKER_PE_VER=$BEAKER_PE_VER
+PUPPET_GATLING_SCALE_BASE_INSTANCES=$PUPPET_GATLING_SCALE_BASE_INSTANCES
+PUPPET_GATLING_SCALE_ITERATIONS=$PUPPET_GATLING_SCALE_ITERATIONS
+PUPPET_GATLING_SCALE_INCREMENT=$PUPPET_GATLING_SCALE_INCREMENT
+PUPPET_GATLING_SCALE_SCENARIO=$PUPPET_GATLING_SCALE_SCENARIO
+ABS_AWS_MOM_SIZE=$ABS_AWS_MOM_SIZE
+============================================"
+}
+
+# Link scale results to nginx content location
+function link_results () {
+    # test name is arg1, pe_ver is arg2
+    source=$HOME/gplt/$1/gatling-puppet-load-test/results/scale
+    dest=/usr/share/nginx/html/gplt/scale/$1/$2
+
+    sudo mkdir -p "${dest}"
+    sudo ln -s "${source}" "${dest}"
+}
+
+# Prepare checkout of puppet-gatling-load-test
+function prep_gplt () {
+    cd "$HOME/gplt" || exit 1
+    mkdir "$1"
+    cd "$1" || exit 1
+    git clone git@github.com:puppetlabs/gatling-puppet-load-test
```

Comment:
instead of creating a dir and putting gatling under it, why not just clone the gatling repo to the dir you are creating directly.

Reply:
Updated to clone into the parameterized directory name directly without created doubly nested directory structure.